### ISSUE TITLE
Shift + Ctrl + key

### ISSUE THREAD

**Issue by guakamole** [Guake/guake#1534]
```
Debian GNU/Linux buster/sid
Gnome: 3.30.2

Guake Terminal: 3.4.0
VTE: 0.54.2
VTE runtime: 0.54.2
Gtk: 3.24.5
zsh: 5.7.1

keycode 166 = XF86Back
keycode 167 = XF86Forward
```

Hello, I recently updated Guake to 3.4.0 and my keybinding `Shift` + `Ctrl` + `Back` (used to close a tab) stopped working. I tried binding `Shift` + `Ctrl` + `Forward` (or `e` or a series of other keys that were not already in use) to the close action and it doesn't work either.

`Shift` + `Ctrl` + `R` does work for renaming. 

It looks like I can't use any key combination involving `Shift` + `Ctrl` other than the ones predefined in Guake.

Any clue on how to debug this ? Is this guake-specific or could the culprit come from somewhere else ?

Thank you in advance for your help :)

**Comment by mati865** [comment]
Probably duplicate of https://github.com/Guake/guake/issues/1478

**Comment by mlouielu** [comment]
Could you test it with master `HEAD`? I cannot reproduce on it :(

**Comment by mlouielu** [comment]
I think it is not related to keybinding issue, I get an exception when I use accel-key to close the tab (v3.4.0), as duplicate of #1478 which @mati865 mention.

```
TypeError: delete_page_current() takes from 1 to 2 positional arguments but 5 were given
```

And it should be fixed by 4a117df, please take a test on 3.5.0 or master `HEAD`
